src/log.h

```diff
@@ -78,73 +80,86 @@ class Logger {
 
 class ILogOutput {
 public:
-	virtual void log(const std::string &line) = 0;
+	virtual void log_raw(LogLevel, const std::string &line) = 0;
```

Comment:
the parameter name isn't needed, it was overtaken from other parts in the file, where SN has omitted it. I think i've asked him that same question before, and he said that the type already says everything. A string is very generic and can mean alot, but a LogLevel in a method called log?

Ah, yes, will rename to logRaw.